### PR TITLE
workflows: Run OS and cloud suites for the JP5 Jetson devices

### PR DESCRIPTION
**Pull request by acostach**
HUP will not work, since these need takeover from the existing production images, so that will be tested manually for migration.

We also want to deploy to staging only for the time-being, if any deploys are made.

Changelog-entry: workflows: Run OS and cloud suites for the JP5 Jetson devices

### CODE REVIEW

**Comment by acostach** [comment]
Autokit tests aren't started

File: .github/workflows/jetson-xavier-nx-devkit-emmc.yml [hunk 1]
```diff
@@ -9,6 +25,11 @@ on:
...
         type: boolean
         default: false
+      deploy-environment:
+        description: Environment to use for build and deploy
+        required: false
+        type: string
+        default: balena-staging.com
 
 permissions:
   id-token: write # This is required for requesting the JWT #https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-amazon-web-services#requesting-the-access-token
```
Comment (rcooke-warwick) [inline .github/workflows/jetson-xavier-nx-devkit-emmc.yml:35]: I think we need to add:

```
actions: read
contents: read
packages: read
id-token: write # This is required for requesting the JWT #https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-amazon-web-services#requesting-the-access-token
pull-requests: write
```

e.g https://github.com/balena-os/balena-alliance-raspberrypi3/pull/473/commits/4af9184d60686b818190646ad21b9fc26b067a90

Comment (rcooke-warwick) [reply .github/workflows/jetson-xavier-nx-devkit-emmc.yml:35]: To all of the workflow files

Comment (acostach) [reply .github/workflows/jetson-xavier-nx-devkit-emmc.yml:35]: thanks @rcooke-warwick , they seem to be present though on all 3 workflow files already

**Pull request merged**